xml/favicons.xml

```diff
@@ -75,10 +75,21 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:apache:tomcat:-"/>
   </fingerprint>
 
-  <fingerprint pattern="^(?:b6341dfc213100c61db4fb8775878cec|cf2445dcb53a031c02f9b57e2199bc03)$">
+  <fingerprint pattern="^(?:b6341dfc213100c61db4fb8775878cec|cf2445dcb53a031c02f9b57e2199bc03|fe22dd2bb09daccf58256611ac491469)$">
     <description>Drupal CMS</description>
+    <!-- Original Drupal favicon https://github.com/drupal/drupal/blob/f0a16bf2a4d1524aa33b656533e37d977cca4802/core/misc/favicon.ico -->
+
```

Comment:
These extra lines have been added by `bin/recog_cleanup` - https://github.com/rapid7/recog/blob/0975a6c1840ccd163ff0beb3832677fb5a314831/bin/recog_cleanup#L11

It looks a bit out of place to me 😄

Reply:
It's likely because most of the comment blocks are used to separate groupings of fingerprints. I'll have to see about additional customization in the cleanup changes.